### ISSUE TITLE
Add classic line charts

### ISSUE THREAD

**Issue by deeplook** [dask/distributed#4132]
As nice as the existing dynamic widgets are, in certain cases it would be helpful to have classic line charts instead of the shiny dynamic bar charts. This would allow tracking changes better over time and take a screenshot for documentation. An example use-case is when debugging memory leak issues like described in https://stackoverflow.com/questions/64046973/dask-memory-leakage-issue-with-json-and-requests/64059237#64059237. I have attached the GIF used there here as well.

![](https://i.stack.imgur.com/AEOgz.gif)

**Comment by TomAugspurger** [comment]
Agreed that this would be useful. I don't believe that this information is currently tracked, however. So we would need to collect it before being able to display it.

**Comment by mrocklin** [comment]
The Bokeh chart itself could also track it if we wanted to.
